src/olympia/yara/tests/test_admin.py

```diff
@@ -0,0 +1,94 @@
+import json
+
+from django.contrib.admin.sites import AdminSite
+from django.utils.html import format_html
+
+from olympia import amo
+from olympia.amo.tests import (TestCase, addon_factory, user_factory,
+                               version_factory)
+from olympia.amo.urlresolvers import reverse
+from olympia.yara.admin import YaraResultAdmin
+from olympia.yara.models import YaraResult
+
+
+class TestYaraResultAdmin(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.user = user_factory()
+        self.grant_permission(self.user, '*:*')
+        self.client.login(email=self.user.email)
+        self.list_url = reverse('admin:yara_yararesult_changelist')
+
+        self.admin = YaraResultAdmin(model=YaraResult, admin_site=AdminSite())
+
+    def test_list_view(self):
+        response = self.client.get(self.list_url)
+        assert response.status_code == 200
+
+    def test_has_add_permission(self):
+        assert self.admin.has_add_permission(request=None) is False
+
+    def test_has_delete_permission(self):
+        assert self.admin.has_delete_permission(request=None) is False
+
+    def test_has_change_permission(self):
+        assert self.admin.has_change_permission(request=None) is False
+
+    def test_formatted_addon(self):
+        addon = addon_factory()
+        version = version_factory(
+            addon=addon,
+            channel=amo.RELEASE_CHANNEL_LISTED
+        )
+        r = YaraResult(version=version)
```

Comment:
We typically avoid one-letter variables in favor of something more descriptive - in that case, I'd go for `result` (same for other instances of `r` below)

Reply:
argh, right. Sorry for that.